Neos.Neos/Classes/Routing/FrontendNodeRoutePartHandler.php

```diff
@@ -182,6 +201,13 @@ protected function convertRequestPathToNode($requestPath)
         if ($requestPathWithoutContext === '') {
             $node = $siteNode;
         } else {
+            if (!empty($this->options['uriPathSuffix'])) {
```

Comment:
I would probably want to extract that into a separate method but can be done later ofc.

Reply:
good idea, extracted it